cluster.go

```diff
@@ -123,6 +134,7 @@ func NewCluster(clusterAddrs []string, opts ...ClusterOpt) (*Cluster, error) {
 	defaultClusterOpts := []ClusterOpt{
 		ClusterPoolFunc(DefaultClientFunc),
 		ClusterSyncEvery(5 * time.Second),
+		ClusterWithTrace(trace.ClusterTrace{}),
```

Comment:
This is unnecessary, since a new Cluster will already have `ct` set to an empty `ClusterTrace`.

Reply:
I'll fix it.